### PR TITLE
Add additional information to README

### PR DESCRIPTION
**Pull request by lzchen**

### CODE REVIEW

File: sdk/monitor/azure-monitor-opentelemetry-exporter/README.md [hunk 1]
```diff
@@ -383,6 +383,11 @@ gauge = meter.create_observable_gauge("gauge", [observable_gauge_func])
 #### Metric custom views
 
 ```python
+"""
+This example shows how to customize the metrics that are output by the SDK using Views. Metrics created
+and recorded using the sdk are tracked and telemetry is exported to application insights with the
+AzureMonitorMetricsExporter.
+"""
```
Comment (jeremydvoss) [inline sdk/monitor/azure-monitor-opentelemetry-exporter/README.md:390]: It seems some examples have these explanation comments IN CODE while others have explanation beforehand. I like the latter. What do you think?

File: sdk/monitor/azure-monitor-opentelemetry-exporter/README.md [hunk 2]
```diff
@@ -420,6 +425,8 @@ my_counter.add(100)
 
 ```
 
+More examples with the metrics `Views` SDK can be found [here](https://github.com/open-telemetry/opentelemetry-python/tree/main/docs/examples/metrics/views).
+
 #### Metric record attributes
 
 ```python
```
Comment (jeremydvoss) [inline sdk/monitor/azure-monitor-opentelemetry-exporter/README.md:432]: Example of code without comment

File: sdk/monitor/azure-monitor-opentelemetry-exporter/README.md [hunk 3]
```diff
@@ -309,11 +309,11 @@ The following sections provide several code snippets covering some of the most c
...
 * [Recording instruments with attributes](#metric-record-attributes)
 
+Review the [OpenTelemetry Metrics SDK][ot_metrics_sdk] to learn how to use OpenTelemetry components to collect metrics.
+
 #### Metric instrument usage
 
 ```python
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License.
 """
```
Comment (jeremydvoss) [inline sdk/monitor/azure-monitor-opentelemetry-exporter/README.md:317]: Example of code with comment

**Review by jeremydvoss — APPROVED** [review]

File: sdk/monitor/azure-monitor-opentelemetry-exporter/README.md (hunk 1 above)
Comment (lzchen) [reply sdk/monitor/azure-monitor-opentelemetry-exporter/README.md:398]: Fixed to be consistent.

**Pull request merged**